zerver/tests/test_message_notification_emails.py

```diff
@@ -1619,8 +1619,26 @@ def test_long_term_idle_user_missed_message(self) -> None:
         hamlet = self.example_user("hamlet")
         othello = self.example_user("othello")
         cordelia = self.example_user("cordelia")
+        zulip_realm = get_realm("zulip")
+
+        # user groups having less than 12 members are small user groups.
+        small_user_group = check_add_user_group(
+            zulip_realm, "small_user_group", [hamlet, othello, cordelia], acting_user=None
+        )
+
+        large_user_group_members: List[UserProfile] = []
+        for count in range(12):
+            member = do_create_user(
+                f"email {count}",
+                f"password {count}",
+                zulip_realm,
+                f"dummy name {count}",
+                acting_user=None,
+            )
```

Comment:
Creating a dozen users is kinda expensive -- this will make this test take a long time to run, which slows the overall test suite. I think a better way to do this test would be to just make the 12 value something you can override for the duration of the test; `settings.MAX_GROUP_SIZE_FOR_MENTION_REACTIVATION`; you can define it in `default_settings.py` and just lower it to 2 for the part of the test where you want the group to be "large".  You can define it just after `MAX_STREAM_SIZE_FOR_TYPING_NOTIFICATIONS` in `default_settings.py`, and not include it in `prod_settings_template.py`, since this isn't a thing we expect anyone to change.

It's more readable to define parameters this way, in addition to better testing flexibility using `with override_settings()`.

Reply:
Makes sense. Made the changes.